cmd/storage.go

```diff
@@ -91,15 +109,27 @@ var lsBlobers = &cobra.Command{
 	Args:  cobra.MinimumNArgs(0),
 	Run: func(cmd *cobra.Command, args []string) {
 		doJSON, _ := cmd.Flags().GetBool("json")
+		doActive, _ := cmd.Flags().GetBool("active")
+
 		var list, err = sdk.GetBlobbers()
 		if err != nil {
 			log.Fatalf("Failed to get storage SC configurations: %v", err)
 		}
-		if doJSON {
-			util.PrintJSON(list)
-			return
+
+		if doActive {
+			if doJSON {
+
+				util.PrintJSON(filterActiveBlobbers(list))
+				return
+			}
+			printBlobbers(filterActiveBlobbers(list))
+		} else {
+			if doJSON {
+				util.PrintJSON(list)
+				return
+			}
+			printBlobbers(list)
```

Comment:
sorry, for being too picky.
could it be done on "data" layer, not "presentation" layer.  not spitting `print` function between scenarios. also lets drop excessive code.

```
if doActive {
    list = filterActiveBlobbers(list);
} 

if doJSON {
    util.PrintJSON(list)
} else {
    printBlobbers(list)
}
```